src/pudl/output/epacems.py

```diff
@@ -4,22 +4,11 @@
 from pathlib import Path
 
 import dask.dataframe as dd
-import pandas as pd
-import sqlalchemy as sa
 
 import pudl
 from pudl.settings import EpaCemsSettings
 
 
-def epacamd_eia(pudl_engine: sa.engine.Engine) -> pd.DataFrame:
-    """Pull the EPACAMD-EIA Crosswalk table."""
-    pt = pudl.output.pudltabl.get_table_meta(pudl_engine)
-    crosswalk_tbl = pt["epacamd_eia"]
-    crosswalk_select = sa.sql.select(crosswalk_tbl)
-    crosswalk_df = pd.read_sql(crosswalk_select, pudl_engine)
-    return crosswalk_df
```

Comment:
I'm surprised the original function didn't apply any type conversions. I wonder if passing the table through `apply_pudl_dtypes` will break downstream analyses.

Reply:
Well, we'll know shortly!